README.md

```diff
@@ -1,10 +1,8 @@
 Puppet Catalog Diff Viewer
 ===========================
 
-[![License](https://img.shields.io/github/license/camptocamp/puppet-catalog-diff-viewer.svg)](https://github.com/camptocamp/puppet-catalog-diff-viewer/blob/master/LICENSE)
-[![Docker Pulls](https://img.shields.io/docker/pulls/camptocamp/puppet-catalog-diff-viewer.svg)](https://hub.docker.com/r/camptocamp/puppet-catalog-diff-viewer/)
-[![Build Status](https://img.shields.io/travis/camptocamp/puppet-catalog-diff-viewer/master.svg)](https://travis-ci.org/camptocamp/puppet-catalog-diff-viewer)
-[![By Camptocamp](https://img.shields.io/badge/by-camptocamp-fb7047.svg)](http://www.camptocamp.com)
```

Comment:
can you add that back? we usualy keep a badge from the original authors like this:
```
[![Donated by Camptocamp](https://img.shields.io/badge/donated%20by-camptocamp-fb7047.svg)](#transfer-notice)
```

Reply:
see #52